NuGet.Config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think this got added by accident. Removing this will also fix the build.

Reply:
Fixed in a39b3d74ea785cefa6d2b5af2b53d143092a20af